File: Constructorio_Net/src/models/SearchResponse.java
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

/**
 * Constructor.io Search Response
 **/
namespace Constructorio_NET
{
    public class SearchResponse {

        [JsonPropertyName("result_id")]
        public String resultId;

        [JsonPropertyName("response")]
        public SearchResponseInner response;

        [JsonPropertyName("request")]
        public Map<String, Object> request;

    }
}

File: src/constructor.io/models/Search/SearchResponse.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

/**
 * Constructor.io Search Response
 */
namespace Constructorio_NET.Models.Search
{
    public class SearchResponse
    {
        [JsonProperty("result_id")]
        public string ResultId { get; set; }

        [JsonProperty("response")]
        public SearchResponseInner Response { get; set; }

        [JsonProperty("request")]
        public Dictionary<string, object> Request { get; set; }
    }
}